Ansible/group_vars/identity.dev

```diff
@@ -21,7 +21,7 @@ server_organization_unit: Grid
 organisation:
   name: INFN
   logo: infn
-#ldap_server:
+ldap_server: vm03.ct.infn.it
```

Comment:
this should in principle be done with a lookup in the task instead of a declared variable. 
Instead of writing {{ ldap_server }} we should have hostvars[group['ldap-servers'][0]][ansible_fqdn]